templates/openstack/architecture.html

```diff
@@ -0,0 +1,265 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1i8rjvhw3N2gDA5NXuKEPcftXY95zePjsURmQu4dSgGo
+{% endblock meta_copydoc %}
+
+{% block title %}OpenStack architecture{% endblock %}
+
+{% block meta_description %}
+  OpenStack under the hood: explore OpenStack architecture and understand how it works.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>OpenStack architecture</h1>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            The best recipe for a successful cloud deployment is to follow a proven design guide from an experienced and trusted advisor. Canonical's reference architecture for OpenStack has been successfully deployed across hundreds of customers and thousands of sites.
+          </p>
+        </div>
+        <hr class="is-muted" />
```

Comment:
There's a vanilla pr about to land that will deprecate this usage, but since this is going into a feature branch, I would rather update and address one that's live on vanilla (although you can do it now instead if you prefer). Either way just wanted to flag it so we don't forget